lib/thumbs.rb

```diff
@@ -7,13 +7,51 @@
 require 'erb'
 require 'netrc'
 require 'http'
-
+require "graphql/client"
+require "graphql/client/http"
+require 'net/http'
+require 'active_support'
 
 require 'thumbs/general_helpers'
 require 'thumbs/webhook_helpers'
 require 'thumbs/pull_request_worker'
 
 
+
+
+module Thumbs
+  module GitHub
+    def self.dump(file)
+      GraphQL::Client.dump_schema(Thumbs::GitHub::HTTP, file)
+    end
+
+    def self.load(file)
+      GraphQL::Client.load_schema(file)
+    end
+  end
+end
+
+module Thumbs
+  module GitHub
+    HTTP = GraphQL::Client::HTTP.new("https://api.github.com/graphql") do
+      def headers(context)
+        {
+            "Authorization" => "Bearer #{ENV['GITHUB_ACCESS_TOKEN']}"
+        }
+      end
+    end
+    file='schema.json'
+    dump(file) unless File.exists?(file)
```

Comment:
Yeh we may want to just check in the schema.json, while also perhaps improving upon this import code. for now it'll just use the one it loaded first. Thanks!